Makefile

```diff
@@ -62,33 +70,56 @@ check_venv:
 	@if [ ! -d "./venv" ]; then python3 -m venv venv; fi
 
 
-pdm_deps: check_venv
+pip_pdm: check_venv
+	@$(venv); pip install pdm
```

Comment:
uh no. Don't pip install PDM...
Point user to https://github.com/pdm-project/pdm#installation

Reply:
You mean, he may want to have it installed by himself and not forced upon him by this ``Makefile`` ?
(as opposed to *don't install it with pip*, because according to the doc, it's legit).

Reply:
"Otherwise, it is recommended to install pdm in an isolated environment with pipx:

pipx install pdm
Or you can install it under a user site:

pip install --user pdm"

Reply:
but this will clash with having installed it by the reccomended method, so I would avoid.

Reply:
I was thinking that ``pip install pdm`` would install inside the user's account or in the enclosing venv, if any.